BD-1433: Link Shorting EA (#3698)
braze-inc/braze-docs: _docs/_hidden/private_betas/link_shortening.md

---
nav_title: Link Shortening

> permalink: "/user_guide/message_building_by_channel/sms/campaign/link_shortening"

Reviewer: ```suggestion
permalink: "/user_guide/message_building_by_channel/sms/campaign/link_shortening"
```
I tried viewing the doc, but the [deploy link](https://braze-docs-ev5u3meeu-braze.vercel.app/docs/user_guide/message_building_by_channel/sms/campaign/link_shortening) is returning a 404 error 🤔

Author: fixed!

hidden: true
---

> # Link Shortening

Reviewer: ```suggestion
# Link shortening
```
One more minor suggestion that I missed in my review! 👍

Author: Hey Lydia! Going to skip this one since I'm pretty sure Link Shortening is the name of the feature, like Canvas or Currents. :)

![][3]

## Testing

We always recommend that you preview and test your message before launching a campaign or Canvas. 

Navigate to the **Test** tab to preview and send an SMS to [content test groups](https://www.braze.com/docs/user_guide/administrative/app_settings/developer_console/internal_groups_tab#content-test-groups) or an individual user. The preview will update with relevant personalization and the shortened URL. The number of characters and [billable segments](https://www.braze.com/docs/user_guide/message_building_by_channel/sms/campaign/segments/) will also update to reflect the rendered personalization and the shortened URL. 

Make sure to save the campaign or Canvas before sending a test message to receive the shortened URL that will be dispatched in your message. If the campaign or Canvas is not saved before a test send, the test send will contain a placeholder URL.

![][2]

{% alert note %}
Liquid personalization and shortened URLs are templated on the **Test** tab after a user has been selected. Ensure a user is selected to receive an accurate character count!
{% endalert %}

## Click tracking

> When Link Shortening is enabled, the SMS and MMS performance table include a column titled **Total Clicks** that shows a count of click events per variant and an associated click rate. For more details on SMS metrics, see [SMS message performance](https://www.braze.com/docs/user_guide/message_building_by_channel/sms/sms_campaign_analytics/#message-performance).

Reviewer: ```suggestion
When Link Shortening is enabled, the SMS and MMS performance table include a column titled **Total Clicks** that shows a count of click events per variant and an associated click rate. For more details on SMS metrics, see [SMS message performance]({{site.baseurl}}/user_guide/message_building_by_channel/sms/sms_campaign_analytics/#message-performance).
```
